third_party/2and3/pymysql/cursors.pyi

```diff
@@ -29,7 +29,7 @@ class Cursor:
     def fetchmany(self, size: Optional[int] = ...) -> Union[Optional[Gen], List[Gen]]: ...
     def fetchall(self) -> Optional[Tuple[Gen, ...]]: ...
     def scroll(self, value: int, mode: Text = ...) -> None: ...
-    def __iter__(self) -> Optional[Gen]: ...
+    def __iter__(self) -> Iterator[Tuple[Gen, ...]]: ...
```

Comment:
Removing `Optional` is correct. But why the change to a tuple? As far as I can see, each iteration returns one row, i.e. the return value of `fetchone()`, which is just `Gen`.

Reply:
Makes sense, it is just `Gen`. Thank you!

Reply:
Okay, I see why I did this. `SSCursor.__iter__` return type was incorrect, I just copied it. I just changed it to `Iterator[Gen]`, as well as `SSCursor.fetchall_unbuffered` return type.